ssh/Dockerfile

```diff
@@ -6,4 +6,6 @@ ENV USER_NAME=mock
 ENV USER_PASSWORD=TLQuoLMn*T89&Y*r*YqHviSFH6MkR!4E
 
 ADD gen.sh /home/mock/gen.sh
-RUN /home/mock/gen.sh 50 > /home/mock/example.txt
\ No newline at end of file
+RUN /home/mock/gen.sh 50 > /home/mock/example.txt &&\
+    echo '* * * * * /home/mock/gen.sh 10 >> /home/mock/example.txt' > /etc/crontabs/root
+CMD /usr/sbin/crond -l 2 -f
```

Comment:
oo i jest cronik, super!
Sprawdźcie jeszcze proszę, co z tym skryptem, że nie działało u Szymona. Imo trzeba sprawdzić, czy skrypt się dobrze odpala na każdym programie (bash, sh)

Reply:
Jeśli chodzi o ten błąd, to na tą chwilę nie mam pomysłu o co może chodzić. Skrypt działa i w bashu i w sh.

Reply:
Poprosimy Szymona o dodanie chmoda. Na tę chwilę zamykamy